RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,120 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+string LX_getDesiredWorkshed(){
+	string currentWorkshed = get_property("auto_workshed").to_lower_case();
+	//return the actual item name in case a shorthand is used
+	switch(currentWorkshed)
+	{
+		case "model train set":
+		case "train":
+			return "model train set";
+		case "cold medicine cabinet":
+		case "cmc":
+			return "cold medicine cabinet";
+		case "diabolic pizza cube": //unsupported
+		case "pizza": //unsupported

```

Comment:
Pizza should get combined with the unsupported stuff. Or at least not return the item as we don't want a workshed that won't get used installed.